### PR TITLE
[StyleCop] Fix all warnings in Bot.Builder.TestBot.WebApi project

### PR DESCRIPTION
**Pull request by Maihav**
•	Add/remove spacing
•	Add element's documentation
•	Add Header to files
•	Convert the files encoding to UTF-8 with byte order mark
•	Match file and type name

### CODE REVIEW

File: tests/Microsoft.Bot.Builder.TestBot.WebApi/App_Start/BotConfig.cs [hunk 1]
```diff
@@ -1,13 +1,20 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
+﻿// <copyright file="BotConfig.cs" company="PlaceholderCompany">
```
Comment (GasparAcevedoZainSouthworks) [inline tests/Microsoft.Bot.Builder.TestBot.WebApi/App_Start/BotConfig.cs]: This is not the correct copyright format. The former  (MIT License) is the correct one.

File: tests/Microsoft.Bot.Builder.TestBot.WebApi/App_Start/UnityConfig.cs [hunk 2]
```diff
@@ -1,6 +1,6 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
+﻿// <copyright file="UnityConfig.cs" company="PlaceholderCompany">
```
Comment (GasparAcevedoZainSouthworks) [inline tests/Microsoft.Bot.Builder.TestBot.WebApi/App_Start/UnityConfig.cs]: Same here.

File: tests/Microsoft.Bot.Builder.TestBot.WebApi/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -1,8 +1,12 @@
-﻿using System.Reflection;
+﻿// <copyright file="AssemblyInfo.cs" company="PlaceholderCompany">
```
Comment (GasparAcevedoZainSouthworks) [inline tests/Microsoft.Bot.Builder.TestBot.WebApi/Properties/AssemblyInfo.cs]: Same here.

Comment (Maihav) [reply tests/Microsoft.Bot.Builder.TestBot.WebApi/Properties/AssemblyInfo.cs]: Fixed

File: tests/Microsoft.Bot.Builder.TestBot.WebApi/App_Start/UnityConfig.cs (hunk 2 above)
Comment (Maihav) [reply tests/Microsoft.Bot.Builder.TestBot.WebApi/App_Start/UnityConfig.cs]: Fixed

File: tests/Microsoft.Bot.Builder.TestBot.WebApi/App_Start/BotConfig.cs (hunk 1 above)
Comment (Maihav) [reply tests/Microsoft.Bot.Builder.TestBot.WebApi/App_Start/BotConfig.cs]: Fixed

**Comment by Aliandi** [comment]
Promoted to MS #1457

**Pull request closed without merging**